packages/core/package.json

```diff
@@ -54,6 +54,7 @@
     "usePathForSuiteName": "true"
   },
   "devDependencies": {
+    "@stellarjs/abstract-transport": "^0.10.1",
```

Comment:
no circular deps here?

Reply:
only needed for tests...acceptable or not?

Reply:
only needed for tests...acceptable or not?

Reply:
i think you get a lerna warning, and not sure if it does anything bad

Reply:
nah nothing bad -approve me pls